contributions/compare.ts

```diff
@@ -0,0 +1,24 @@
+import { Targeting } from "../types";
+
+interface CompareMeta {
+  targeting: Targeting;
+  expectedTest: string;
+  expectedVariant: string;
+}
+
+const apiURL =
+  "https://contributions.guardianapis.com/epic/compare-variant-decision";
+
+// Temporary helper to evaluate our variant logic server-side
+// No useful response expected (so discard).
```

Comment:
Since we're treating this as fire-and-forget, do we want to stick a `catch` on the end so we don't get unhandled promise rejection warnings if something _does_ go wrong?